1-exercises/E-array-map/exercise.js

```diff
@@ -3,7 +3,7 @@
 
 let numbers = [0.1, 0.2, 0.3, 0.4, 0.5];
 
-let numbersMultipliedByOneHundred; // complete this statement
+let numbersMultipliedByOneHundred = numbers.map(*100);
```

Comment:
Please check this to find more about the map method
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map
https://www.w3schools.com/jsref/jsref_map.asp
https://www.freecodecamp.org/news/javascript-map-how-to-use-the-js-map-function-array-method/